src/reducers/playground.ts

```diff
@@ -0,0 +1,34 @@
+import { Action, Reducer } from 'redux'
+import { IUpdateEditorValue, UPDATE_EDITOR_VALUE } from '../actions/playground'
+
+/**
+ * A state for the playground container
```

Comment:
In general, since we have established a convention, I think the comment can be terser and only to describe non-obvious name or to explain a complex routine/function.

I think we can go with no comment by default, the reviewer will be the one responsible for requesting documentation if deemed necessary, what do you think?